docs/src/requirements/child-devices.md

```diff
@@ -0,0 +1,76 @@
+
+# Child Device Concept
+
+TODO: Add motivation for child-device concept. Somethin like:<br/>
+-- Device-side and Cloud-side structural view (that might match the physical structure)<br/>
+-- Features are available per child-device (e.g. Software Management with one SW list per child device, ...)
+
+That document summarises a collection of use-cases and requirements, based on experiences of Sofware AG and some customers/partners.
+
+## Definition
+1) A child device is a physical device connected to the thin-edge device (typically a gateway device)  or even over IP.
+2) Generally there are two kinds of child devices:
+   - a) A dumb device that is connected to the thin-edge device via an IO-Link or fieldbus, that can't run the processing logic that sends/receives data to/from the thin-edge device. For such devices, the processing logic will be running on the thin-edge device itself.
+   - b) A smart device that is connected to the thin-edge device over an IP network, with the processing logic running on that device itself. Its interactions with the thin-edge device will only be over the network
+3) In both kinds of devices, there will always be a "child device process" that is responsible for handling all data to/from the physical child device.
+4) Thin-edge's responsibility is to receive and route any cloud-bound data from these child device processes and any device-bound data like software/config binaries to these child device processes.
+5) The child device process itself is resposible for pushing these binaries to the physical device itself and applying it there.
+6) Each child device has a unique ID that can be used to identify/distinguish between them
+
+TODO: Add somehow statement below as definition: 
+"PLCs are already long time in production-line and no additional software can be installed (e.g. MQTT client) or configuration changed, to make the PLC capbable to communicate with the thin-edge device. That is to avoid any risc in the production line. Instead the thin-edge device must use existing interfaces from the PLC."
```

Comment:
```suggestion
"On PLCs that are already running in production-line, no new software can be installed (e.g. an MQTT client, or a new C application) or configurations changed, to make the PLC communicate with the thin-edge device directly. Instead, the thin-edge device must use existing interfaces from the PLC to interact with it. That is to avoid any risks in the production line."
```